build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 // please update RELEASE_NOTES.md when you update this version
-def buildVersion = "2.2.4"
+def buildVersion = "2.2.5"
```

Comment:
[nit] Looking at [semver](https://semver.org/) think this would be a minor version (add functionality in a backward compatible manner) rather than a patch (make backward compatible bug fixes)?